tester/orb.yaml

```diff
@@ -153,6 +153,11 @@ jobs:
         description: |
           Services to build besides target service
         type: string
+      build_args:
```

Comment:
Perhaps a bit confusing, since `--build-args` is already an option/concept using `docker-compose build`. How about using `build_options` instead? 

```shell
$ docker-compose build -h
Usage: build [options] [--build-arg key=val...] [--] [SERVICE...]
```

Reply:
I don't know if I would want to change `compose_args` because that would require both releasing the change as a major release and updating all dependents to use the new notation.

I see your point about calling them `build_options`. The parameter `build_args` is now `build_options`.

Reply:
Yeah, doesn't seem worthy of a major update as it stands. Perhaps can leave as a `TODO` for the next major version update? I am always in favor of consistency :) That being said, I don't think it is confusing as is.